src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -254,7 +254,7 @@
     <comment>Context menu item</comment>
   </data>
   <data name="ContextMenuDelete" xml:space="preserve">
-    <value>Remove</value>
+    <value>Delete</value>
```

Comment:
In general - please don't combine different tasks/improvements into one PR.

Reply:
Ok, will separate this out into its own PR.

Reply:
https://github.com/DynamoDS/Dynamo/pull/12088